firmware/controllers/engine_controller.cpp

```diff
@@ -558,6 +558,9 @@ bool validateConfigOnStartUpOrBurn() {
 		ensureArrayIsAscending("Fuel CLT mult", config->cltFuelCorrBins);
 		ensureArrayIsAscending("Fuel IAT mult", config->iatFuelCorrBins);
 
+		ensureArrayIsAscending("Boost CLT mult", config->cltBoostCorrBins);
```

Comment:
or should it be ensureArrayIsAscendingOrDefault? we have poor and inconsistent approach towards compatible changes

and we can only afford a cheap solution here at this point

Reply:
Now I use `ensureArrayIsAscendingOrDefault` function instead of `ensureArrayIsAscending`